src/webview/static/README.md

```diff
@@ -0,0 +1,15 @@
+# Webview static files
+
+This directory holds static files to be packaged for use by the embedded
+WebView. Files herein (other than those named `README.md`) will be copied out to
+the packaged bundle at build time, into a subdirectory named `webview`.
```

Comment:
New version looks good, thanks!

The other information in your reply is interesting and sounds useful, and I was going to suggest putting it in comments in appropriate places. Then I saw that the info about the iOS root seems to already be in a comment on `assetsPath` in one of your later commits, which looks like a perfect spot.

For the point about keeping things consistent: I think that means consistency between `assetsPath`; the paths passed at the call sites of `tools/build-webview`; and the validation in the latter script itself. Is that right?

Given your point about Xcode :stuck_out_tongue: , perhaps that comes down to the script's own validation, and `assetsPath` and/or other app code that relies on the paths. I think a pointer at least from the script to `assetsPath` would be helpful -- I know when I was looking at the mention of `file:///android_asset/webview` yesterday I was curious where that path came from, or what it has to be consistent with.

Reply:
Added a few pointers in the relevant locations. (I've somewhat arbitrarily decided that `tools/build-webview` gets to be the "owner" of this decision: it seems a bit too fiddly to go in the current README, which is vaguely targeted at people producing assets rather than people maintaining the build system. That could be changed, though.)

Reply:
Looks good!

> I've somewhat arbitrarily decided that `tools/build-webview` gets to be the "owner" of this decision: it seems a bit too fiddly to go in the current README

Sounds good to me.